tests/test_loadReferenceCatalog.py

```diff
@@ -0,0 +1,204 @@
+import unittest
+import copy
+
+import astropy.units as u
+
+import lsst.utils.tests
+import lsst.afw.geom
+import lsst.pipe.base as pipeBase
+from lsst.meas.algorithms import LoadReferenceObjectsTask, getRefFluxField, ReferenceObjectLoader
+from lsst.pipe.tasks.loadReferenceCatalog import LoadReferenceCatalogConfig, LoadReferenceCatalogTask
+from lsst.pipe.tasks.colorterms import Colorterm, ColortermDict, ColortermLibrary
+
+
+synthTerms = ColortermLibrary(data={
+    "synth*": ColortermDict(data={
+        "filter1": Colorterm(primary="ref1", secondary="ref2", c0=0.0, c1=0.01),
+        "filter2": Colorterm(primary="ref2", secondary="ref3", c0=0.0, c1=-0.01),
+    })
+})
+
+
+_synthFlux = 100.0
+_synthCenter = lsst.geom.SpherePoint(30, -30, lsst.geom.degrees)
+
+
+def setup_module(module):
+    lsst.utils.tests.init()
+
+
+# class TrivialLoader(LoadReferenceObjectsTask):
```

Comment:
Does this comment still need to be here?

Reply:
This was leftover from earlier code, and should be removed.  Thanks!